docs/robots/ridgeback/maintenance_ridgeback.mdx

```diff
@@ -4,9 +4,117 @@ sidebar_position: 4
 ---
 
 import ComponentIntroductionRidgeback from "/components/introduction_ridgeback.mdx";
+import GettingNewPackages from "/components/maintenance/getting_new_packages.mdx";
+import Support from "/components/support.mdx";
 
 <ComponentIntroductionRidgeback />
 
-:::warning todo—under construction
+## Battery Replacement
+
+If Ridgeback's batteries need replacement, they are accessible by removing the top-plate and the insulator
+cover. Before performing any service or maintenance to the robot, the battery pack must be fully disconnected.
+The batteries weigh approximately 33 kg (73 lbs) each, so use all necessary care and caution when lifting the
```

Comment:
suggest removing 'all necessary' from the sentence.

Reply:
The Antenna cables also need to be carefully removed.

Reply:
Made both updates